src/public/modules/users/controllers/examples-card.client.directive.js

```diff
@@ -99,9 +101,9 @@ function examplesCardController(
       controllerAs: 'vm',
       resolve: {
         FormToDuplicate: () => {
-          return FormApi.template({
-            formId: $scope.form._id,
-          }).$promise.then((res) => res.form)
+          return $q
+            .when(FormApi.queryTemplate($scope.form._id))
```

Comment:
I don't think you need $q.when here either. the uibmodal [docs](https://angular-ui.github.io/bootstrap/) say that this works the same way as `resolve` in ui-router.